docs/en/setup/backend/backend-receivers.md

```diff
@@ -157,19 +157,19 @@ receiver_jaeger:
 
 NOTICE, Jaeger receiver is only provided in `apache-skywalking-apm-x.y.z.tar.gz` tar.
 
-## Opencensus receiver
+## OpenTelemetry receiver
 
-Opencensus receiver supports to ingest agent metrics by meter-system. OAP can load the configuration at bootstrap. 
-If the new configuration is not well-formed, OAP fails to start up. The files are located at `$CLASSPATH/oc-rules`.
+OpenTelemetry receiver supports to ingest agent metrics by meter-system. OAP can load the configuration at bootstrap. 
+If the new configuration is not well-formed, OAP fails to start up. The files are located at `$CLASSPATH/otel-rules`.
 
 The file is written in YAML format, defined by the scheme described in [prometheus-fetcher](./backend-fetcher.md).
-Notice, `receiver-oc` only support `metricsRules` node of scheme due to the push mode it opts to.
+Notice, `receiver-otel` only support `metricsRules` node of scheme due to the push mode it opts to.
 
-To active the `default` implementation:
+To active the `oc` implementation which receives metrics the [OpenCensus exporter](https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/opencensusexporter/README.md) pushes:
 ```yaml
-receiver-oc:
-  selector: ${SW_OC_RECEIVER:-}
-  default:
+receiver-otel:
+  selector: ${SW_OC_RECEIVER:oc}
+  oc:
```

Comment:
Could we consider to make `oc` as an attribute? I have a feeling, we need `oc` and `otel` up and running at the same time, in the future.

Reply:
done